mlflow/openai/utils/chat_schema.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from typing import Any
+
+from mlflow.entities import SpanType
+from mlflow.entities.span import LiveSpan
+from mlflow.exceptions import MlflowException
+from mlflow.tracing import set_span_chat_messages, set_span_chat_tools
+
+_logger = logging.getLogger(__name__).setLevel(logging.DEBUG)
```

Comment:
```suggestion
_logger = logging.getLogger(__name__)
```
debug log should only be shown when users enable it (tho I guess you just forget to remove this after debugging)

Reply:
oops yeah i enabled this for testing and forgot to remove lol, thanks for catching